metadata/recreation/ski_area_boundaries.md

```diff
@@ -0,0 +1,71 @@
+# Title
+
+Utah Ski Area Boundaries
+
+## ID
+
+9dbabdee-ae86-4715-86e9-f4dba518d596
+
+## Brief Summary
+
+Polygon dataset of approximate ski resort areas in Utah.
+
+## Summary
+
+This dataset contains polygon boundaries for all ski resorts in Utah. This dataset contains geographic boundaries only and does not include pricing, availability, or other information about the ski resorts.
+
+## Description
+
+### What is the dataset?
+
+This dataset includes the following ski area resorts: Alta, Beaver Mountain, Brian Head, Brighton, Cherry Peak, Deer Valley, Eagle Point, Nordic Valley, Park City, Powder Mountain, Snowbasin, Snowbird, Solitude, and Sundance. The boundaries are simplified for general cartographic or location purposes and do not depict exact areas of operation, avalanche control, or ownership.
+
+### What is the purpose of the dataset?
+
+This dataset has been made available for general analytic and cartographic purposes.
+
+### What does the dataset represent?
+
+Polygons in this dataset represent approximate geographical boundaries for ski resorts in Utah. Each feature includes the name of the ski area and a ‘four color’ field for symbolization.
+
+### How was the dataset created?
+
+UGRC created this dataset by digitizing points on top of high resolution aerial photography and terrain models. These boundaries are current as of 2016-2017 season. We maintain this dataset and update it as needed.
```

Comment:
Ideally we shouldn't keep dates in the metadata as they become technical debt. The update history should be current in the update history sheet and come from a separate area in the metadata system.

Reply:
Good point. That's been removed!